modules/plugin/coverage-multidim/netcdf/src/test/java/org/geotools/coverage/io/netcdf/NetCDFMosaicReaderTest.java

```diff
@@ -136,6 +136,7 @@
                 AbstractGridFormat.USE_JAI_IMAGEREAD.createValue();
         imageRead.setValue(false);
         NO_DEFERRED_LOADING_PARAM = imageRead;
+        System.setProperty("user.timezone", "UTC");
```

Comment:
Is it being re-set at the end of the test? Surefire JVMs are reused across modules, setting a sysvar can affect a test downstreams in an inpredictable way.

Reply:
Nope. I suspect I can remove that set anyway.
It was there for a previous mvn run error reporting a different time even if running it on IntelliJ or through surefire debug it didn't show up.

Let's see how the tests go without that property (removing it right now).